1880 Paragraph after bullet list not working as expected

millianapia opened the pull request.

fernandolucchesi commented on web/components/src/List/List.tsx:
```diff
@@ -15,6 +15,7 @@ const StyledList = styled(EdsList)<ListProps>`
   margin-left: var(--space-medium);
   margin-right: var(--space-medium);
   padding-left: var(--space-medium);
+  padding-bottom: var(--space-small);
```
Adding the padding to the List component itself will have a colateral to all these components:

![image](https://github.com/equinor/energyvision/assets/23337266/edf73481-0dae-431b-8698-d4d7d1a03459)

A solution like the one implemented for News is more suitable for this

fernandolucchesi replied:
Adding the padding to the List component itself will have a colateral to all these components:

![image](https://github.com/equinor/energyvision/assets/23337266/edf73481-0dae-431b-8698-d4d7d1a03459)

A solution like the one implemented for News is more suitable for this

millianapia commented:
Refactored richtext and ingress so the list will have a margin, instead of wrapping each use. Also removed the text size - this might be a problem for some components and need to be applied directly where it should be used instead of directly in the ingress component

fernandolucchesi commented on web/pageComponents/shared/portableText/IngressText.tsx:
```diff
@@ -9,7 +9,7 @@ import { Text, List } from '@components'
 const { Item } = List
 
 const StyledList = styled(List)`
-  font-size: var(--typeScale-2);
```
does it make sense to remove it for all use cases? maybe it was intentional for ingress?

fernandolucchesi commented on web/pageComponents/shared/portableText/RichText.tsx:
```diff
@@ -44,8 +44,16 @@ const defaultSerializers = {
   list: {
-    bullet: BulletList,
-    number: NumberedList,
+    bullet: ({ children }: PortableTextBlock) => (
```
new code is the same as before, but more verbose. better to have it as it was before

fernandolucchesi commented on web/pageComponents/topicPages/Promotion.tsx:
```diff
@@ -12,7 +12,7 @@ const Wrapper = styled.div`
 `
 const Ingress = styled.div`
-  margin-bottom: var(--space-xLarge);
+  margin-bottom: var(--space-Large);
```
hmm why this has to be changed?

millianapia commented on web/pageComponents/topicPages/TextBlock.tsx:
```diff
@@ -98,7 +101,7 @@ const TextBlock = ({ data, anchor }: TextBlockProps) => {
         {text && (
           <TextContainer>
-            <RichText value={text} />
+            <RichText value={text}></RichText>
```
ill fix this so it will be back to />

fernandolucchesi commented on web/pageComponents/topicPages/TextBlock.tsx:
```diff
@@ -39,6 +39,9 @@ const StyledTextBlock = styled.section`
     margin-bottom: 0;
   }
 `
+const StyledList = styled(List)`
```
this is unused

millianapia replied:
is now removed

fernandolucchesi commented on web/pageComponents/shared/portableText/components/BulletList.tsx:
```diff
@@ -1,5 +1,10 @@
+import styled from 'styled-components'
+
+const StyledList = styled(List)`
+  margin-bottom: var(--space-medium);
```
I think this is the best solution given the current setup we have (in which I hate - adding margin bottom to elements instead of gaps to the parent div). However, we need to keep in mind that for use cases we could end up with double margins. Example, an ingress that applies margin-bottom, and ends with a styled list. Nothing to change here, but just to keep in mind that these can occur, then we fix case by case when reported
